rbac/management/role/view.py

```diff
@@ -70,8 +70,15 @@ def application_filter(self, queryset, field, values):
             query = query | Q(access__permission__istartswith=f"{application}:")
         return queryset.distinct().filter(query)
 
+    def permission_filter(self, queryset, field, values):
+        """Filter to lookup role by application(s) in permissions."""
+        permissions = values.split(",")
+
+        return queryset.filter(access__perm__in=permissions).distinct()
```

Comment:
So this is what we'd update when the refactor to use `Permission` happens?

Reply:
Right, after the permission association, we only have to refactor here, which is simple 😁